app/src/components/domain/RiskImminentEvents/Gdacs/index.tsx

```diff
@@ -133,21 +229,26 @@ function Gdacs(props: Props) {
                 ? footprint_geojson
                 : undefined;
 
-            const geoJson: GeoJSON.FeatureCollection<GeoJSON.Geometry> = {
+            const geoJson: GeoJSON.FeatureCollection<GeoJSON.Geometry, EventGeoJsonProperties> = {
                 type: 'FeatureCollection' as const,
                 features: [
                     ...footprint?.features?.map(
                         (feature) => ({
                             ...feature,
                             properties: {
-                                ...feature.properties,
-                                type: getLayerType(feature.geometry.type),
+                                eventName: feature?.properties?.eventname,
```

Comment:
We need an `id` field on all feature for the layer to be interactive. Let's make sure that is the case.